Source/WebCore/css/parser/CSSParserFastPaths.cpp

```diff
@@ -47,7 +47,7 @@
 
 namespace WebCore {
 
-bool CSSParserFastPaths::isSimpleLengthPropertyID(CSSPropertyID propertyId, bool& acceptsNegativeNumbers)
+bool CSSParserFastPaths::isSimpleLengthPropertyID(CSSPropertyID propertyId, ValueRange& valueRange)
```

Comment:
It would be a bit more idiomatic for this to return a std::optional. Maybe define a type like:

```
struct SimpleLengthProperties {
    ValueRange valueRange;
};

std::optional<SimpleLengthProperties> CSSParserFastPaths::simpleLengthPropertiesForPropertyID(CSSPropertyID propertyId) 
{
...
}
```